apps/minifront/src/state/status.ts

```diff
@@ -10,9 +10,15 @@ export interface StatusSlice {
 
 export const createStatusSlice = (): SliceCreator<StatusSlice> => set => ({
   start: async () => {
-    const stream = viewClient.statusStream({});
-
     try {
+      //set the current status using unary request, since statusStream does not guarantee that we will get the status
+      const status = await viewClient.status({});
+      set(state => {
+        state.status.fullSyncHeight = status.fullSyncHeight;
+        if (!status.catchingUp) state.status.latestKnownBlockHeight = status.fullSyncHeight;
```

Comment:
Brackets on this plz

Reply:
Not sure we need `if (!status.catchingUp)`, do we?

Reply:
> Not sure we need `if (!status.catchingUp)`, do we?

This is not necessary, but it allows us to display that we are fully synchronized before we get the first message from the `statusStream` 
Also interesting that the `StatusStream` documentation says `Stream sync status updates until the view service has caught up with the chain`

https://github.com/penumbra-zone/penumbra/blob/2c9c3f3694440e651325306f2345f74796285157/proto/penumbra/penumbra/view/v1/view.proto#L29-L31

In our implementation, streaming continues even after we've caught up with chain (and I think it's the right thing to do) 
But theoretically, there could be other view service implementations where streaming just doesn't happen when we catch up with the chain